sdk/storage/azure-storage-blobs/inc/azure/storage/blobs.hpp

```diff
@@ -1,12 +1,21 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // SPDX-License-Identifier: MIT
 
+/**
+ * @brief Includes all public headers from Azure Storage Blobs SDK library.
+ *
+ */
+
 #pragma once
 
+#include "azure/storage/blobs/protocol/blob_rest_client.hpp"
+
```

Comment:
```suggestion

```

Reply:
This is internal implementation and should not be exposed directly to customers.

Reply:
How is this relevant to "Doxygen comment updates"?

Reply:
I was making sure that the description of the file ("Includes all public <XXX> SDK library headers") corresponds to what it actually does. I.e. if a header has stuff defined in non-`_detail` or non-`_internal` namespace, it has public stuff. I did not look whether it is included by other headers in the same library or not - because today it might be included, but tomorrow maybe not.

But if you don't like the changes to the list of the include-all header, it's fine, I'll undo them, no problem. It is your lib to update. Just wanted to help. Less than anything do I want to break something in your code, or to dictate how you should do minor things.